Fix Tasks API Schema Mismatch Issue [sc-67784]

keith-chartmogul opened the pull request:
This PR fixes an issue with the Tasks API schema, specifically ensuring that the `task_uuid` property is returned in task responses instead of `uuid`. Resolves #112.

briwa reviewed (approved):
LGTM. I've also made a small edit to the PR description. See [this doc](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue)

<details><summary>Details</summary>
<p>

![Screenshot 2025-05-19 at 11 21 36 AM](https://github.com/user-attachments/assets/a42007fa-f6b2-42f1-af53-ce3155e43cfb)


</p>
</details>

briwa commented:
Also @keith-chartmogul you might want to bump the version in this PR too, similar to how you did it in #109, i.e. b17ef741ed0fbb455f51a31afc3e61c8d682d51e

geopet85 reviewed (approved):
Minor but why raise the version to 4.6.2 since 4.6.1 was not tagged or released to pypi? Perhaps we can combine altogether?

keith-chartmogul commented:
> Also @keith-chartmogul you might want to bump the version in this PR too, similar to how you did it in #109, i.e. [b17ef74](https://github.com/chartmogul/chartmogul-python/commit/b17ef741ed0fbb455f51a31afc3e61c8d682d51e)

Good call, I updated the version number to `4.7.0` and adjusted the changelog as well. And thanks for adjusting the PR description to automatically close the issue raised by George.

@briwa @geopet85 Pushing another commit does not clear the original approval, so I just want to get a re-approval in case I misunderstood Briwa's suggestion. Thanks for understanding. 🙏

briwa commented:
@keith-chartmogul Let's go with George's suggestion to release it as `4.6.1` (as opposed to what you did `4.7.0`) -- the unreleased code is mostly fixes and improvements IMO, so it should be a patch upgrade instead of a minor upgrade

briwa commented on CHANGELOG.md:
```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning].
 [Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
 [Semantic Versioning]: https://semver.org/spec/v2.0.0.html
 
+## [4.7.0] - 2025-05-19
+- Fixed Tasks API schema issue
+
 ## [4.6.1] - unreleased
 - Unify requirements in a single place
```
```suggestion
## [4.6.1] - 2025-05-19
- Fix Tasks API schema issue
- Unify requirements in a single place
```

geopet85 replied:
This is what I meant. 4.6.1 should be the target version IMO.

keith-chartmogul replied:
Sorry for the confusion guys 🫠

I've updated to @briwa's suggestion above and amended `version.py` as well. No need for any more re-approvals as it is clear what was being asked now. I'll merge the PR as it currently is and release right away. 🫡

briwa reviewed (commented):
All that's left after updating the changelog below is just to push the tag `v4.6.1` and afaik the new tag should be added in pypi as well
